### DISCUSSION TITLE
Support for additional keys

### DISCUSSION THREAD

**Discussion by DannyBen** [charmbracelet/vhs#64] [Ideas]
What is the plan for supporting additional keys?

In particular, I got stuck trying to send the Delete key, but I am hoping there is a plan to add support for more keys.

**Comment by maaslalani** [comment]
Hey, we'll definitely add more keys in the future. They're not too difficult to add. Do you mind creating an issue (or just list here) with all the keys you need (including Delete)

**Comment by DannyBen** [comment]
Sure, will do.
